### PR TITLE
New package: cppcms-1.0.5

### PR DESCRIPTION
**Pull request by asafo**

### CODE REVIEW

File: srcpkgs/cppcms/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+version=1.0.5
+revision=1
+build_style=cmake
+hostmakedepends=""
+makedepends="gnutls-devel pcre-devel python"
+short_desc="C++ Web Framework"
+maintainer="Asaf Ohayon <asaf@sysbind.co.il>"
+license="LGPL-3"
+homepage="http://cppcms.com"
+distfiles="http://netix.dl.sourceforge.net/project/cppcms/cppcms/${version}/cppcms-${version}.tar.bz2"
```
Comment (Vaelatern) [inline srcpkgs/cppcms/template]: Find a way to use `$SOURCEFORGE_SITE` please.

File: srcpkgs/cppcms/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'cppcms'
+pkgname=cppcms
+version=1.0.5
+revision=1
+build_style=cmake
+hostmakedepends=""
```
Comment (Vaelatern) [inline srcpkgs/cppcms/template]: `python` probably goes in here

**Comment by Vaelatern** [comment]
Please rebase on master and force push, to trigger a Travis build. Do not open and close PRs.

**Comment by the-maldridge** [comment]
@asafo If you still wish to keep this PR, please reply here, otherwise it will be closed due to inactivity.

**Pull request closed without merging**